.eslintrc.json

```diff
@@ -28,7 +28,7 @@
         "_nx": true
     },
     "parserOptions": {
-        "ecmaVersion": 8
+        "ecmaVersion": 12
```

Comment:
It means that we drop support of Edge browser < v79. It seems too risky to me in a bugfixes version.

This change should be done in main branch only.

Reply:
Changed to target main. Since targeting main anyways, ES level was raised to 13. I looked at this year's version (ES14) but it was just finalized last month and it looks like at least some features aren't supported in Firefox yet.

ES13 features seem to be implemented in all recent, supported browser versions.
https://caniuse.com/?feats=mdn-javascript_builtins_array_at,mdn-javascript_builtins_regexp_hasindices,mdn-javascript_builtins_object_hasown,mdn-javascript_builtins_error_cause,mdn-javascript_operators_await_top_level,mdn-javascript_classes_private_class_fields,mdn-javascript_classes_private_class_methods,mdn-javascript_classes_static_class_fields,mdn-javascript_classes_static_initialization_blocks

I can backport the individual fixes made in this PR